core/api/src/graphql/error-map.ts

```diff
@@ -731,7 +737,7 @@ export const mapError = (error: ApplicationError): CustomGraphQLError => {
         error.message ? ": " + error.message : ""
       })`
       return new UnknownClientError({ message, logger: baseLogger })
-
+      123
```

Comment:
??

Reply:
if I had a cat I would blame him to walk on the keyboard. but I don't have a cat ^^